src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -161,6 +161,7 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
 | PowerPC      | `reg_nonzero`  | None     | `3`            | `b`           |
 | PowerPC      | `freg`         | None     | `0`            | None          |
 | s390x        | `reg`          | None     | `%r0`          | None          |
+| s390x        | `reg_addr`     | None     | `%r1`          | `a`           |
```

Comment:
`a` is a constraint, not a modifier. Modifiers only affect how a register is formatted to a string. This should just be `None`.

Reply:
Oh, I included it because the PowerPC's `b` constraint (`reg_nonzero`) is there, but they are indeed wrong.
Fixed them.